Packs/ThreatQ/Integrations/ThreatQ_v2/ThreatQ_v2.py

```diff
@@ -146,6 +153,34 @@ def get_errors_string_from_bad_request(bad_request_results, status_code):
     return str()  # Service did not provide any errors.
 
 
+def get_tlp_from_indicator(sources):
+    if not sources:
+        return None
+    tlp = 0
+    for source in sources:
+        try:
+            tlp = int(source.get('TLP')) if int(source.get('TLP')) > tlp else tlp
+        except Exception as e:
+            demisto.debug(str(e))
```

Comment:
Please add a message here "Failed getting TLP from source". If you could also mention what's the `source` name it'll be best.